civis/ml/_model.py

```diff
@@ -37,7 +37,8 @@
 
 # Map training template to prediction template so that we
 # always use a compatible version for predictions.
-_PRED_TEMPLATES = {8387: 8388,  # v1.0
+_PRED_TEMPLATES = {9112: 9113,  # v1.1
+                   8387: 8388,  # v1.0
```

Comment:
Please set this to predict with template 9113; we should use the most up-to-date template possible for the predictions.